MSFragger-GUI/src/com/dmtavt/fragpipe/cmd/CmdMSBooster.java

```diff
@@ -105,8 +106,8 @@ public boolean configure(Component comp,
       return false;
     }
 
-    if (!predictRT && !predictSpectra) {
-      SwingUtils.showErrorDialog(comp, "At least one of <b>Predict RT</b> or <b>Predict spectra</b> must be enabled if you want to run MSBooster.", NAME + " parameter error");
+    if (!predictRT && !predictSpectra && !predictIm) {
+      SwingUtils.showErrorDialog(comp, "At least one of <b>Predict RT</b> or <b>Predict spectra</b> or <b>Predict IM</b> must be enabled if you want to run MSBooster.", NAME + " parameter error");
```

Comment:
Is it meaningful only having `predictIm=true`?

Reply:
> We can enforce that one of spectra or RT is predicted

That is what the original code is doing.

Reply:
added back in latest commit